### PR TITLE
Adds Mime and Clown lockers

### PR DESCRIPTION
**Pull request by uraniummeltdown**
The Clown and Mime now have their own lockers! Lockers come with one set of job equipment in them. Sprites taken from Hippie but edited to work on Paradise.

:cl:
add: Added mime and clown lockers to the game
add: Added mime and clown lockers to their offices
:cl:

### CODE REVIEW

**Comment by IcyV** [comment]
Personally I'm :-1: for the extra banana seed in the locker, but I like everything else.

**Comment by Fox-McCloud** [comment]
They're ok, sprite wise, but I'm really not sure why such a low-level job needs a *secure* locker.

**Comment by Lady-Luck** [comment]
Yeaah, low level jobs shouldn't really need secure lockers. Seems silly to me.

**Comment by Fox-McCloud** [comment]
>They're not really that secure

Uh, what. Anyone who doesn't have access to the the locker can't get it open--that's, by definition *secure*.

IMO, just remove the access on these, so they're still a thematic locker, but they're not necessarily *ID locked*---similar to how freezer's are.

**Comment by Doukhan** [comment]
Chaplain barman the chef botany all have secure lockers, civilians aside pretty much everyone does form cargo techs to surgeons, the clown/mime are the odd ones out really.

They'd make a great place to stash bodies too.

**Comment by pinatacolada** [comment]
the clown locker better goddamn honk when i interact with it

File: code/game/objects/structures/crates_lockers/closets/secure/miscjobs.dm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	name = "clown's locker"
+	req_access = list()
+	icon_state = "clownsecure1"
+	icon_closed = "clownsecure"
+	icon_locked = "clownsecure1"
+	icon_opened = "clownsecureopen"
+	icon_broken = "clownsecurebroken"
+	icon_off = "clownsecureoff"
+
+	New()
```
Comment (Fox-McCloud) [inline code/game/objects/structures/crates_lockers/closets/secure/miscjobs.dm]: No relative pathing.

File: code/game/objects/structures/crates_lockers/closets/secure/miscjobs.dm [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	name = "mime's locker"
+	req_access = list()
+	icon_state = "mimesecure1"
+	icon_closed = "mimesecure"
+	icon_locked = "mimesecure1"
+	icon_opened = "mimesecureopen"
+	icon_broken = "mimesecurebroken"
+	icon_off = "mimesecureoff"
+
+	New()
```
Comment (Fox-McCloud) [inline code/game/objects/structures/crates_lockers/closets/secure/miscjobs.dm]: No relative pathing.

**Comment by uraniummeltdown** [comment]
>the clown locker better goddamn honk when i interact with it

I'd love to add this but I'm not sure how to code this, not very experienced

**Comment by Brad3733** [comment]
Why wouldn't you want the locker to be secure? Anyone in any profession has somewhere they deem safe and secure to store their equipment, whether you're a Clown, Cleaner or Bin man, it makes no difference. Not giving Clown's and Mime's secure lockers because they're "low level jobs" is a ridiculous excuse.

**Pull request merged**